rpaas/manager.py

```diff
@@ -171,7 +171,10 @@ def bind(self, name, app_host):
                 return
             if bound_host is not None:
                 raise BindError("This service can only be bound to one application.")
-        self.consul_manager.write_location(name, "/", destination=app_host)
+        empty_upstream = False
+        if router_mode:
+            empty_upstream = True
+        self.consul_manager.write_location(name, "/", destination=app_host, empty_upstream=empty_upstream)
```

Comment:
Wouldn't using `empty_upstream=router_mode` here be the same?

Reply:
Ouch!